src/main/java/dev/jbang/cli/Cache.java

```diff
@@ -34,6 +36,7 @@ public Integer clear(
 			classes.addAll(Arrays.asList(Settings.CacheClass.values()));
 		} else if (urls == null
 				&& jars == null
+				&& jbang == null
```

Comment:
Minor bit but technically it's jbangs not jbang right ?

Reply:
Well, no, technically it's just a single one. That's why I used singular, although it goes against my ingrained love for symmetry 😂

Reply:
If you want I can make it plural of course.

Reply:
symmetry ftw :) please update and i'll happily merge :)

Reply:
Done

Reply:
Hmm not yet

Reply:
Ok, now :-)